plugins/inputs/kernel/kernel.go

```diff
@@ -30,25 +30,39 @@ var (
 type Kernel struct {
 	statFile        string
 	entropyStatFile string
+	ksmStatsDir     string
+
+	optCollect map[string]bool
+
+	ConfigCollect []string `toml:"collect"`
```

Comment:
Please move this to the top of the struct as this is where you can usually find the options...